commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDisplayLayoutLocalServiceImpl.java

```diff
@@ -114,12 +116,25 @@ public CPDisplayLayout deleteCPDisplayLayout(Class<?> clazz, long classPK) {
 		return null;
 	}
 
+	@Override
+	public void deleteCPDisplayLayoutByG_L(long groupId, String layoutUuid) {
```

Comment:
We shouldn't use the `*ByG_L` as part of the public name here -- we always make this friendlier. It looks like we could just call it `deleteCPDisplayLayout` (as long as that signature with `(long, String)` isn't already taken). If not, we'd probably call it `deleteCPDisplayLayoutByLayoutUuid`.